src/solc_0.7/catalyst/AssetAttributesRegistry.sol

```diff
@@ -113,6 +113,7 @@ contract AssetAttributesRegistry is WithMinter, WithUpgrader {
     /// @param assetId id of the asset
     /// @param gemIds list of gems ids to set
     function addGems(uint256 assetId, uint16[] calldata gemIds) external {
+        // TODO: verify audit comment
```

Comment:
Is this function could be simplified ?

Reply:
probably, do you want to break into part?
Not sure it is too bad though

Reply:
Actually there's nothing serious to simplify.